DEVELOPMENT.md

```diff
@@ -8,6 +8,10 @@ To contribute to the Tozny Platform Admin JS SDK, review these guidelines carefu
 
 Code linting is provided by ES Lint, and coding style is managed with Prettier. Ensure all code is lint free and formatted code before submitting it to the Tozny Platform Admin JS SDK.
 
+In addition to automated linting, some things should be kept in mind when designing new APIs:
```

Comment:
💜 for the thought and the effort to document this

Reply:
all the props go to @rosasck for suggesting it